esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -1914,7 +1914,7 @@ void WsWuInfo::getWorkunitEclAgentLog(const char* fileName, const char* agentPid
     }
 
     if (buf.length() < 1)
-        buf.append(47, "(No logfile entries found for this workunit)");
+        buf.append("(No logfile entries found for this workunit)");
```

Comment:
As far as I can tell this will append the chars and a null character at the end. The other calls to buf.append (passing in length) will not be including the null character at the end.

Not sure what is intended though.